tests/MongoDB.Bson.Tests/Serialization/Serializers/BsonDocumentBackedClassSerializerTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿/* Copyright 2018-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System.Collections.Generic;
+using MongoDB.Bson.Serialization;
+using MongoDB.Bson.TestHelpers;
+
+namespace MongoDB.Bson.Tests.Serialization.Serializers
+{
+    public class BsonDocumentBackedClassSerializerTests
```

Comment:
We didn't have any tests for this before either. But this is the file were the BsonDocumentBackedClassSerializerReflector needs to live.

I can write tests for this abstract base class if you want. It's implicitly tested by the tests for ChangeStreamDocumentSerializer.

Reply:
Implicit tests sound sufficient to me.